tests/cucumber/features/login.feature

```diff
@@ -1,5 +1,18 @@
 Feature: Log in with CheckIn A person with an identity can log in with CheckIn
+  Background: I start on the home page
+    Given I am a test user 
+    And I am on the main page
+  
   Scenario: The login button is present
     Given I visit the main page
     When I look for the login button
     Then it is present
+
+  Scenario: A test user visits the forum to log in
+    Given I click on the login button
+    Then I am taken to EGI AAI
+    Then I choose SSO
```

Comment:
In the current IdP Discovery page there is no dedicated "EGI SSO" login button so the user needs to use the search list. In the upcoming version there will also be a login button. Would it make sense to test both methods (i.e. testing if the login button-that-will-be-available-really-soon-now exists, as well as searching for it)?

Reply:
We want to make sure that we understand user behaviour with these tests. Currently, the behaviour is to type something in the search box and see what the filtered list looks like. 
Implemented in the following way : 
```
@browser.text_field(id: 'username', name: 'j_username').set @username.to_s
```
If you add a new "EGI SSO" button, we should have a slightly different scenario where we write the feature and test it. 
I think the best thing is to first write the feature in [Gherkin](https://docs.cucumber.io/gherkin/) - how do you want users to behave?